docs/src/app/components/pages/components/left-nav.jsx

```diff
@@ -165,6 +208,13 @@ export default class LeftNavPage extends React.Component {
               <MenuItem index={0}>Menu Item</MenuItem>
               <MenuItem index={1}><a href="/link">Link</a></MenuItem>
             </LeftNav>
+            <LeftNav open={this.state.navOpen} menuItems={menuItems} />
+            <LeftNav 
+              open={this.state.undockedNavOpen} 
+              onNavOpen={this._showLeftNavUndockedControlledClick}
```

Comment:
:scream: yeah it must be removed, and warnings about them must be added. I'll do that too :+1: